skimage/transform/_warps.py

```diff
@@ -6,6 +6,14 @@
                          _convert_warp_input, _clip_warp_output)
 
 
+def _multichannel_default(multichannel, ndim):
+    # utility for maintaining previous color image default behavior
+    if ndim == 3:
+        return True
+    else:
+        return False
```

Comment:
The tests are set up to fail on warnings so that those don't slip by unnoticed, but if you look at some of the other tests, you'll see that we have utility functions (``skimage._shared._warnings``) that can take care of it.